### ISSUE TITLE
Raise default values

### ISSUE THREAD

**Issue by sturmundbraem** [xperseguers/t3ext-image_autoresize#112]
We've had serious issues due to installing image_autoresize without thinking about changing the default values. 
So the resize defaults `768px` and `1024px` kicked in, and we just discovered this after hundreds of images had been uploaded by FE users and were saved in a small format.

We've fixed this for us for the future, but I think it might make sense to raise these default values eg. to 5000px.
Making smaller is always possible, but larger not.

**Comment by xperseguers** [comment]
No, 5000px is not a sane default. I agree 768 and 1024 is possibly a bit low nowadays, but the purpose of this extension is to work out-of-the-box. That's unfortunate you didn't pay attention, sorry for you.

**Comment by sturmundbraem** [comment]
> That's unfortunate you didn't pay attention

I'm not blaming you for it, of course! It's our bad. I'm just thinking about protecting others from the same dumb mistake. wdy think is a sane default size? I used to put 3000 in the last years, but in this specific project we use the photos for print as well, that's why I went up to 5000.
